### PR TITLE
Remove the SB routes from on-http

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
Jenkins: depends on https://github.com/RackHD/RackHD/pull/897, https://github.com/RackHD/on-taskgraph/pull/290

File: static/monorail-2.0.yaml [hunk 1]
```diff
@@ -3762,6 +3709,7 @@ paths:
         type: string
       responses:
         200:
+
```
Comment (geoff-reid) [inline static/monorail-2.0.yaml]: remove blank line

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Pull request merged**